dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/imports/validation/TrackedEntityImportValidationTest.java

```diff
@@ -292,7 +292,7 @@ void shouldFailToUpdateWhenUserHasAccessToRegistrationUnitAndTEWasTransferred()
     assertNoErrors(trackerImportService.importTracker(new TrackerImportParams(), trackerObjects));
     trackerObjects = fromJson("tracker/validations/enrollments_te_enrollments-data_2.json");
     TrackerImportParams trackerImportParams = new TrackerImportParams();
-    trackerImportParams.setUserId(USER_10);
+    trackerImportParams.setUserId(USER_8);
```

Comment:
hard to understand this change 😅 what makes this user "special"? could you describe it in a javadoc on the constant?

Reply:
That's the problem of having all the test configuration in a json file.
What makes this user special is the org units in its search scope. I could add a javadoc, but if for any reason we change the org units of this user, I'm pretty sure we'll forget about the javadoc and it won't be consistent anymore.
To me, it would be better rely less on json and configure the setup programatically, that way it makes it's easier to understand settings like this one.